packages/browser-destinations/src/destinations/heap/trackEvent/__tests__/index.test.ts

```diff
@@ -36,15 +36,49 @@ describe('#trackEvent', () => {
         type: 'track',
         name: 'hello!',
         properties: {
-          banana: '📞'
+          banana: '📞',
+          apple: [
+            {
+              carrot: 12,
+              broccoli: [
+                {
+                  onion: 'crisp',
+                  tomato: 'fruit'
+                }
+              ]
+            },
+            {
+              carrot: 21,
+              broccoli: [
+                {
+                  tomato: 'vegetable'
+                },
+                {
+                  tomato: 'fruit'
+                }
+              ]
+            }
+          ]
```

Comment:
This change feels a little risky; I'd like to see a few more test cases. Let's add an empty array, a boolean, a float point, a null value and an array of flat properties.